src/registrar/models/domain.py

```diff
@@ -718,7 +718,7 @@ def map_epp_contact_to_public_contact(
 
         if not isinstance(contact, eppInfo.InfoContactResultData):
             raise ContactError("Contact must be of type InfoContactResultData")
-
+        # temp comment for push - will remove
```

Comment:
Does this comment still need to be removed?

Reply:
Nice catch, thank you!